NERC-CEH/plankton_ml
# Set things straight after the change to src layout

* Fully moves the code that got left behind in merge conflicts into the `src` package layout
* Changes the test/lint workflows to run in the same way as [dri-cicd](https://github.com/NERC-CEH/dri-cicd/) - `ruff` for lint, pip install for the test environment
* Adds a separate install from `apt` for `exiftool` to the workflow, notes about doing this rootless to the README
* Unpins `chromadb` as the issues causing test failures are mitigated 
* Removed some lingering direct imports of `scivision` and updated the scripts that build an embedding collection to reflect those changes

### To test

Checked this with both a fresh conda `env` and fresh `venv` (discrepancy between local and Actions runs was the chroma version pinning in `pyproject.toml` which we thankfully no longer need)

Please see the README for walkthrough and remember to install `exiftool` dependency if you don't already have it (I'm _sure_ it was available on my managed VM when I first wrote the header-embedding code...)
